service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/Descriptor.java

```diff
@@ -390,14 +390,14 @@ public String toString() {
     /**
      * Describes a topic call.
      */
-    public static final class TopicCall<Message> {
+    public static final class TopicCall<MESSAGE> {
```

Comment:
In any case, there are lots of type parameters in Lagom using the current convention, and this is just changing one. I think it would be best to revert it from this pull request so we can merge it, and then if you'd like to continue the discussion, maybe we can have it on the mailing list until there's a consensus.